### DISCUSSION TITLE
New version release 2022?

### DISCUSSION THREAD

**Discussion by Anywhere-Music-Player** [MonitorControl/MonitorControl#1040] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Is there any version release in 2022?

In releases it's 2021 latest.. seems like really outdated. 

### Steps to reproduce

Open releases and no new releases.

### Expected behavior

App updates on weekly/monthly basis.

### Environment Information (please complete the following information)

```markdown
macOS.
```

**Comment by waydabber** [comment]
Well, we actually did 2-3 years worth of feature updates last year (went from 2.0 to 4.0 within 4 months with loads of new features, just check out the release history). :)

But a v4.1.0 release is planned which will be more of a service release that will include all the pull requests since 4.0.2.
